Language/Functions/Time/millis.adoc

```diff
@@ -64,7 +64,7 @@ void loop() {
 
 [float]
 === Notes and Warnings
-Please note that the return value for millis() is of type `unsigned long`, logic errors may occur if a programmer tries to do arithmetic with smaller data types such as `int`. Even signed `long` may encounter errors as its maximum value is half that of its unsigned counterpart.
+Please note that the return value for millis() is of type `unsigned long`, logic errors may occur if a programmer tries to do arithmetic with smaller data types such as `int`. Even signed `long` may encounter errors as its maximum value is half that of its unsigned counterpart. millis() uses timer0's overflow interrupt.
```

Comment:
I think it's a great idea to document this. However, I have a question:
Is this true for all architectures officially supported by Arduino?:
- [AVR](https://github.com/arduino/ArduinoCore-avr)
- [megaAVR](https://github.com/arduino/ArduinoCore-megaavr)
- [SAM](https://github.com/arduino/ArduinoCore-sam)
- [SAMD](https://github.com/arduino/ArduinoCore-samd)
- [MBED](https://github.com/arduino/ArduinoCore-nRF528x-mbedos)
- [ARC32](https://github.com/arduino/ArduinoCore-arc32)

We need to be careful to not assume that Arduino == AVR because it's much more than that. 

That said, we don't need to worry about covering 3rd party cores in the Arduino Language Reference. It is the responsibility of their authors to document where there are differences from the official Arduino documentation.

Reply:
Thanks for your feedback, sorry for my late reply. Indeed this warning seems limited to AVR, megaAVR and SAM(D) architectures, as
-ArduinoCore-arc32 uses (but does not reprogram) TIMER0 which is running at 32MHz,
-ArduinoCore-mbed uses a lowPowerTimer if supported, or a Timer otherwise, and there is no limit on the number of such timers

I rewrote the remark, and would welcome your feedback.